scripts/deploy_assisted_service.sh

```diff
@@ -73,6 +73,113 @@ elif [ "${DEPLOY_TARGET}" == "ocp" ]; then
 
     wait_for_url_and_run "$SERVICE_BASE_URL" "spawn_port_forwarding_command $SERVICE_NAME $OCP_SERVICE_PORT $NAMESPACE $NAMESPACE_INDEX $OCP_KUBECONFIG ocp $CLUSTER_VIP $SERVICE_NODEPORT"
     print_log "${SERVICE_NAME} can be reached at ${SERVICE_BASE_URL}"
+elif [ "${ENABLE_KUBE_API}" == "true" ]; then
```

Comment:
not sure it's good to reuse the name of this env-var
it's doing a bit differently from the original deployment method